README.md

```diff
@@ -31,9 +31,37 @@ curl -X POST localhost:3000 -H 'content-type: application/json' -d '{"hello": "w
 ## Routing and route-handlers
 In lieu of pre-set middleware, routes handlers can be arrays of functions that will
 be iterated over asynchronously. To simplify handling of these handlers,
-it is expected that the handlers will return then-ables.  This means any promises
-library should work (including the built in one), as well as using the `async` function
-keyword.
+it is expected that the handlers will return then-ables, or terminate the response
+stream.  This means any promises library should work (including the built in one),
+as well as using the `async` function keyword.
+
+**If you do not return a then-able, handler processing will stop in that function**
```

Comment:
When you say "then-able" do you mean Promise? "then-able" might not be the best way to describe this.